tracer/src/Datadog.Trace/Debugger/LiveDebuggerFactory.cs

```diff
@@ -51,6 +56,24 @@ public static LiveDebugger Create(IDiscoveryService discoveryService, IRemoteCon
         var probeStatusPoller = ProbeStatusPoller.Create(probeStatusSink, settings);
 
         var configurationUpdater = ConfigurationUpdater.Create(tracerSettings.Environment, tracerSettings.ServiceVersion);
-        return LiveDebugger.Create(settings, serviceName, discoveryService, remoteConfigurationManager, lineProbeResolver, debuggerSink, probeStatusPoller, configurationUpdater);
+
+        IDogStatsd statsd;
+        if (FrameworkDescription.Instance.IsWindows()
+            && tracerSettings.Exporter.MetricsTransport == TransportType.UDS)
+        {
+            Log.Information("Metric probes are not supported on Windows when transport type is UDS");
```

Comment:
Why not? 🤔

Reply:
I would like to but we ran into some problems with it. Maybe in a future PR.